app/src/main/java/de/danoeh/antennapod/fragment/AddFeedFragment.java

```diff
@@ -145,8 +144,8 @@ private void addUrl(String url) {
         startActivity(intent);
     }
 
-    private void performSearch(String query) {
-
+    private void performSearch(EditText searchEditText) {
```

Comment:
Does the text box even need to be passed to the method? I think the method could just access the layout directly without any parameters.

Reply:
Sure, I refactored the code to access the input field directly.